### ISSUE TITLE
[Blasted Lands] Mojo the Twisted (rare) missing movement

### ISSUE THREAD

**Issue by syssneck** [azerothcore/azerothcore-wotlk#7121]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

Original report: [Chromie issue#1286](https://github.com/chromiecraft/chromiecraft/issues/1286)

### Current Behaviour

Mojo the Twisted GUID 134350 (https://wowgaming.altervista.org/aowow/?npc=8296) is stationary.

![mojotwisted](https://user-images.githubusercontent.com/80769357/126891770-3a62e796-e009-4f2f-9ba0-f67c18aa47bd.jpg)

### Expected Blizzlike Behaviour

Mojo should roam a small area around his spawn points.

### Source

World of Warcraft Atlas:
![mojo](https://user-images.githubusercontent.com/80769357/126891763-5d459c27-bba5-4bf1-be18-690a7e972d4c.jpg)

### Steps to reproduce the problem

1. Teleport to Mojo the twisted GUID 134350 (entry 8296)
2. Observe no movement

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/3d4befd7043fbbceb32f7c2a6e0a4f9f638f8b0a

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

Confirmed on local AC (d187ba1).
He does not move, his MovementType is set to 0 in the db:
![unknown_2021 07 26-09 10](https://user-images.githubusercontent.com/68868567/126948129-828259ba-b6d4-4875-9ed2-a477f5524acf.png)

**Comment by gurveleg** [comment]
No, he has several different spawn points. The ledge and hut in top of the tower  are two of them.

**Comment by syssneck** [comment]
Okay, thank you!
